requirements-frozen.txt

```diff
@@ -63,7 +63,6 @@ opentargets-validator==1.0.0
 pandarallel==1.6.4
 pandas==1.5.3
 pathos==0.3.2
-pkg_resources==0.0.0
```

Comment:
Good. I looked at the dependency tree, and apparently this package is not used by anything we are using at the moment.